src/views/TabView/TabBarBottom.js

```diff
@@ -157,6 +164,30 @@ class TabBarBottom extends React.PureComponent<Props> {
       this.props.getTestIDProps && this.props.getTestIDProps(scene);
     return testIDProps;
   };
+  shouldUseHorizontalTabs() {
+    const { routes } = this.props.navigation.state;
+    const {
+      isLandscape,
+      initialLayout,
+      layout,
+      enableHorizontalTabs,
+    } = this.props;
+    const maxTabBarItemWidth = 125;
+    let tabBarWidth;
+    if (layout && layout.width) tabBarWidth = layout.width;
```

Comment:
`layout.width` always exists. The check is not necessary

Reply:
check is not for existence, but for non-zero.  If layout.width is 0, this will return false and I can short-circuit.

Reply:
`layout && layout.width` checks for both falsy `layout` and falsy `width`. `layout` will always be an object and the `width` property will always exit